skills/src/lasr_skills/look_at_person.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import smach_ros
+from geometry_msgs.msg import Point, PointStamped
+import smach
+from vision import Get3DImage
+from lasr_vision_msgs.srv import BodyPixDetection, BodyPixDetectionRequest
+from lasr_vision_msgs.msg import BodyPixMaskRequest
+from lasr_skills import LookToPoint
+import cv2
+import rospy
+from sensor_msgs.msg import Image
+from cv2_img import cv2_img_to_msg, msg_to_cv2_img
+from markers import create_and_publish_marker
+from visualization_msgs.msg import Marker
+from cv2_pcl import pcl_to_img_msg
+import ros_numpy as rnp
+
+DEBUG = True
+
+class CheckEyes(smach.State):
+    def __init__(self):
+        smach.State.__init__(self, outcomes=["succeeded", "failed"], input_keys=["poses", "img_msg", "masks"],
+                             output_keys=["pointstampted"])
+        self.img_pub = rospy.Publisher("/debug/image", Image, queue_size=1)
+        self.marker_pub = rospy.Publisher("eyes", Marker, queue_size=1)
+
+    def execute(self, userdata):
+        # the img msg is pcl
+        if userdata.poses is None:
+            return "failed"
+
+        left_eye, right_eye, eye_point = [], [], []
+
+        for pose in userdata.poses:
+            for keypoint in pose.keypoints:
+                if keypoint.part == "leftEye":
+                    left_eye = keypoint.xy
+                if keypoint.part == "rightEye":
+                    right_eye = keypoint.xy
+
+        if DEBUG:
+            img = msg_to_cv2_img(pcl_to_img_msg(userdata.img_msg))
+
+            cv2.circle(img, (left_eye[0], left_eye[1]), 5, (0, 255, 0), -1)
+            cv2.circle(img, (right_eye[0], right_eye[1]), 5, (0, 255, 0), -1)
+            img_msg1 = cv2_img_to_msg(img)
+
+            self.img_pub.publish(img_msg1)
```

Comment:
I like this, but I think we should pass the `DEBUG` flag to the constructor of the state machine instead of declaring it globally above!